vizro-core/src/vizro/models/_components/container.py

```diff
@@ -14,6 +14,16 @@
 
 
 class Container(VizroBaseModel):
+    """Container component. [`Container`][vizro.models.Container].
+
+    Args:
+        type (Literal["container"]): Defaults to `"container"`.
+        components (List[ComponentType]): See [ComponentType][vizro.models.types.ComponentType]. At least one component
```

Comment:
Is there no validator for this?

Reply:
Validators will be added after Petar implements some changes on the backend. I added it on the to-do list.